src/components/dashboard/components/dictionary/styles/dictionary-modal.css

```diff
@@ -11,3 +11,47 @@
      margin-left: -36%;
      margin-top: 20%  
  }
+ .dictionaryDetails{
+    padding-top: 3%;
+    padding-left: 4%;
+    padding-right: 8%;
+    margin-left: 13%;
+}
+.para{
```

Comment:
You might wanna use meaningful words as CSS class names.

Reply:
i agree

Reply:
That is true

Reply:
Done